### ISSUE TITLE
servers should just run maprotation instead of forcing a specific map

### ISSUE THREAD

**Issue by bmorel** [DaemonEngine/Daemon#552]
Servers apply rotation based on the file indicated by variable `g_initialMapRotation`, but only after the 1st game ended.

The 1st game receive a special treatment: it requires config/server.cfg to end with a `map foobar`, and thus can not receive the rotation custom configuration without duplicating them in config/map/default.cfg.

I would suggest to instead just run the 1st entry of g_initialMapRotation.

**Comment by slipher** [comment]
Yes, it's an awkward situation that you are forced to start with a specific map.

What do you mean by the "rotation custom configuration"?

> I would suggest to instead just run the 1st entry of g_initialMapRotation.
As a workaround, you could emulate this by adding `nextmap` right after the initial `map` command.

**Comment by bmorel** [comment]
> What do you mean by the "rotation custom configuration"?

I mean the per-map configuration which is inside the maprotation file, i.e.: 

```
if ( numPlayers < 24 ) openfield {
  bla
}
```

`bla` can be various things (I'm not sure which ones exactly, but this at least includes bot fill, even if config/map/foo.cfg might be a better place).

**Comment by slipher** [comment]
Well commands in the maprotation is kinda broken anyway, since it won't work if the map is chose by a nextmap vote, for example. Better to use the map configs if possible

**Comment by bmorel** [comment]
One thing they allow is to easily share bulk of the config between multiple servers, unlike the config/maps folder, since that one will also contain the pseudo MotD of the server (the `cp 20 hello world` command) that one will typically want to be rather distinctive.
